tsconfig.json

```diff
@@ -24,7 +24,7 @@
     "strictNullChecks": true,
     "target": "es2016",
     "lib": [
-      "es2017"
+      "es2018"
```

Comment:
This should stay at es2017.  We still need to support node 8.9.4 which doesn't support Promise finally or the regular expression additions (which are really the only two differences between 2017 and 2018).